### DISCUSSION TITLE
Look & Feel - Designator Column in DECODES Platform Tab

### DISCUSSION THREAD

**Discussion by jbatterman** [opendcs/opendcs#737] [General]
I recently was shown (thanks @MikeNeilson) a new option in our profiles that allow you to have a separate column for the Platform designator.  This is the platformListDesignatorCol=true option.  I enabled the option since we are beginning to use designators heavily and I thought it would be easier to scan through things to have it separated.  Without the Designator column enabled, Designators are shown in the Platform column with the syntax of <Site Name>-<Designator>

With the designator tab, the Designators are shown in a separate column.  However, they are still shown under the Platform column as well.  For example, the Platform now equals <Site Name>-<Designator>, and the Designator now equals <Designator>.  This looks a little odd to my eye and repetitive.  My thought is that with the Designator column enabled, that Designator should be removed from the Platform name (Platform = <Site Name>).

An example of what I'm looking at here:
![image](https://github.com/user-attachments/assets/2512caef-15ee-4f10-88c4-81d7633f8a33)

**Comment by MikeNeilson** [comment]
@opendcs/opendcs-core-devs @opendcs/opendcs-triage
